src/layouts/post-layout.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import Footer from "../components/footer";
+import NavBar from "../components/nav-bar";
+import { StyledContainerWrapper } from "../elements/containers";
+
+function Layout({ children, pageContext }) {
+      return (
+          <StyledContainerWrapper>
+            <NavBar />
+            <main>
+              {children}
+            </main>
+            <Footer />
+          </StyledContainerWrapper>
+      )
+}
```

Comment:
Please use always the same type of component to maintaining the consistency
```suggestion
const Layout = ({ children, pageContext }) => (
          <StyledContainerWrapper>
            <NavBar />
            <main>
              {children}
            </main>
            <Footer />
          </StyledContainerWrapper>
      );
```

Reply:
Also some comment about the PropTpes, file extension and index